### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by elsume**
Some spelling mistakes and repeated words that have been fixed.

### CODE REVIEW

**Review by nwalters512 — APPROVED** [review]
Thanks! This will be good to go once you accept the CLA.

**EDIT**: you'll need to resolve the merge conflict before this can land.

**Comment by elsume** [comment]
I have reviewed and hereby sign the CLA

**Pull request merged**
